server/http/controller.go

```diff
@@ -710,6 +710,20 @@ func (c *controller) GetEnvironments(ctx context.Context, take, skip int32, quer
 	}), nil
 }
 
+func (c *controller) GetEnvironmentDefinitionFile(ctx context.Context, environmentId string) (openapi.ImplResponse, error) {
```

Comment:
The controller file is getting quite large, let's consider a refactor to split it up. I like DDD for API backend development

Reply:
+1 to both comments. I think we need to move on from the openapi generator for anything other than entities